flask_monitoringdashboard/database/__init__.py

```diff
@@ -105,8 +75,25 @@ class Outlier(Base):
     time = Column(DateTime)
 
 
+class TestRun(Base):
+    """ Stores unit test performance results obtained from Travis. """
+    __tablename__ = 'testRun'
+    # name of executed test
+    name = Column(String(250), primary_key=True)
```

Comment:
use an integer id as a primary key.
using a string column as a primary key is inviting future pain.

Reply:
Allright, I will change this. Since the functionality that I'm implementing later this week will require additional changes to the database and thus also a migration file, I think it's easiest to change both these things at the same time.